packages/es-components/src/components/containers/horizontalScrollWrapper/HorizontalScrollWrapper.js

```diff
@@ -0,0 +1,225 @@
+import React, { useRef, useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import Icon from '../../base/icons/Icon';
+import withWindowSize from '../../util/withWindowSize';
+
+const OuterWrapper = styled.div`
+  overflow: hidden;
+  position: relative;
+`;
+
+const InnerWrapper = styled.div`
+  alignitems: center;
+  display: flex;
+  height: 100%;
+  position: relative;
+  white-space: nowrap;
+  z-index: 0;
+`;
+
+const ScrollIcon = styled.div`
+  align-items: center;
+  cursor: pointer;
+  display: flex;
+  height: 100%;
+  outline: 0;
+  position: absolute;
+  top: 0;
+  z-index: 1;
+`;
+
+const ArrowContainer = styled.div`
+  align-items: center;
+  background-color: ${props => props.theme.colors.white};
+  border-radius: 50%;
+  box-shadow: 0 1px 1px ${props => props.theme.colors.boxShadowDark};
+  color: ${props => props.theme.colors.gray9};
+  display: flex;
+  padding: 0.5em;
+`;
+
+const generateOnArrowKeyDownHandler = onKeyDown => event => {
+  const code = event.code.toLowerCase();
+  if (code === 'Enter' || code === 'Space') {
+    onKeyDown();
+  }
+};
+
+const ArrowLeft = ({ onClick }) => (
+  <ScrollIcon
+    role="button"
+    tabIndex={0}
+    style={{ left: '0', textAlign: 'left' }}
```

Comment:
That's because `ArrowLeft` and `ArrowRight` need the different values for these (left vs right). I could create two new styled components that add on to what `ScrollIcon` does, but it felt like overkill for something simple like this 🤷. I can do it if there are strong feelings though!

Reply:
I think this is fine for these two elements. I might try the `css` prop over the `style` prop though.

Reply:
Thanks for teaching me about the `css` prop! Pretty cool little trick!